clang/test/VFS/remap-to-fake.c

```diff
@@ -0,0 +1,26 @@
+// RUN: rm -rf %t
+// RUN: mkdir -p %t/From
+// RUN: mkdir -p %t/To
+// RUN: echo '#pragma once' > %t/From/B.h
```

Comment:
Could you rewrite the test using `split-file`? That is much easier to read.

Reply:
Done!